src/Tickets/Commerce/Reports/Orders.php

```diff
@@ -37,21 +37,33 @@ class Orders extends Report_Abstract {
 	public $orders_page;
 
 	/**
-	 * Gets the Orders Report.
+	 * Gets the Orders Report title.
 	 *
-	 * @since 5.2.0
+	 * @since TBD
+	 *
+	 * @param int $post_id
 	 *
 	 * @return string
 	 */
-	public function get_title($post_id) {
+	public function get_title( int $post_id ): string {
```

Comment:
This can be subjective, but I think this could be risky as we're not sure if this one is invoked only with ints, and by setting the parameter type if the function is not invoked with an int is going to give a fatal.